ParkingFinder/handlers/user_information.py

```diff
@@ -41,6 +43,78 @@ def get(self, user_id):
                 'error': "BAD REQUEST Invalid User Id"
             })
 
+    @coroutine
+    def post(self, user_id):
+        # set to default to avoid missingArgumentException
+        access_token = self.get_argument(name='access_token', default=None)
```

Comment:
I tested yesterday, get_argument doesn't work if body is a string. we have to load the string and parse into object. same as the link below
https://github.com/Big-Lemon/ParkingFinder/pull/26/files#diff-c2ad4de5768ba27a98fdd4fbeab793feR132